internal/flags/customflags.go

```diff
@@ -111,6 +111,30 @@ func (f *DirectoryFlag) GetDefaultText() string {
 	return f.GetValue()
 }
 
+// Expands a file path
+// 1. replace tilde with users home dir
+// 2. expands embedded environment variables
+// 3. cleans the path, e.g. /a/b/../c -> /a/c
+// Note, it has limitations, e.g. ~someuser/tmp will not be expanded
+func expandPath(p string) string {
```

Comment:
Not sure what you did here. First I thought you only moved the code to a different location, but you also made some changes? At least the comment about names pipes have been dropped. 

Since you both moved and modified, it becomes difficult to review the change. Please move this back down to where it was previously.